src/toolbar/toolbar-title.jsx

```diff
@@ -60,7 +72,7 @@ const ToolbarTitle = React.createClass({
     }, style);
 
     return (
-      <span style={styles} {...other} >{text}</span>
+      <span className={this.props.className} style={styles} {...other} >{text}</span>
```

Comment:
The class is already inside the `...other` could you extract it using the spread?